proto/defs.bzl

```diff
@@ -32,3 +32,4 @@ proto_lang_toolchain = _proto_lang_toolchain
 # Modules
 proto_common = _proto_common
 ProtoInfo = _ProtoInfo
+toolchains = _toolchains
```

Comment:
I'd prefer not to expose additional symbols in `defs.bzl`. Each symbol should be in a separate bzl file. That reduces transitively loaded bzl files and prevents repo, macro, and rule code from needlessly loading each other.

https://docs.google.com/document/d/1L1JFgjpZ7SrBinb24DC_5nTIELeYDacikcme-YcA7xs/edit

Also, exposing symbol `toolchains` seems to be a very general name.
Also `toolchains` contains a temporary transitional functionality, doesn't it?

Reply:
Let me try to explain more clearly:

In language ruleset implementations like `py_proto_library`, do you expect these three helpers to be available?
https://github.com/bazelbuild/rules_proto/blob/main/proto/proto_common.bzl#L51-L53
- `use_toolchain`
- `find_toolchain`
- `if_legacy_toolchain`

Currently I'm forced to copy-paste some of that logic instead.

Reply:
Note @fmeum as you recommended them yesterday